### PR TITLE
Update lighthouse to v7.3.0

### LINKED ISSUE #580: Lighthouse v7.3.0
Hello guys,

Lighthouse [v7.3.0](https://github.com/GoogleChrome/lighthouse/releases/tag/v7.3.0) was released a 12 days ago.

The release notes includes the `jsonld` update to [its latest version](https://github.com/GoogleChrome/lighthouse/pull/12257)

It fixes this vulnerability described below:

https://npmjs.com/advisories/1650 

image

### PR DESCRIPTION
**Pull request by mattdell**
Fixes #580 (severe vulnerability fix) by updating lighthouse version from 7.0.0 to 7.3.0

### CODE REVIEW

**Comment by mattdell** [comment]
@googlebot I signed it!

**Review by patrickhulce — CHANGES_REQUESTED** [review]
Thanks very much @mattdell! 🎉 

The process for updating the Lighthouse version in this repo though is rather involved and described in the [contributing docs](https://github.com/GoogleChrome/lighthouse-ci/blob/main/CONTRIBUTING.md#updating-the-lighthouse-version). I think in this case with minimal 7.0.0 -> 7.3.0 changes we don't need the full blown version but we need to ensure the `@lhci/utils` and `@lhci/cli` versions match.

File: packages/server/src/ui/routes/build-view/audit-detail/audit-detail-pane.stories.jsx [hunk 1]
```diff
@@ -87,6 +92,15 @@ export const Version700 = () => (
   />
 );
 
+export const Version730 = () => (
+  <AuditDetailPane
+    selectedAuditId={auditPairs700[1].audit.id || ''}
```
Comment (patrickhulce) [inline packages/server/src/ui/routes/build-view/audit-detail/audit-detail-pane.stories.jsx:97]:
```suggestion
    selectedAuditId={auditPairs730[1].audit.id || ''}
```

File: packages/server/test/test-utils.js [hunk 2]
```diff
@@ -92,6 +92,8 @@ function createActualTestDataset() {
       {...baseBuild, id: '7', hash: '1242', commitMessage: 'build 8', runAt: runAt(8)},
       {...baseBuild, id: '8', hash: '1243', commitMessage: 'build 9', runAt: runAt(9)},
       {...baseBuild, id: '9', hash: '1244', commitMessage: 'build 10', runAt: runAt(10)},
+      {...baseBuild, id: '10', hash: '1243', commitMessage: 'build 11', runAt: runAt(9)},
+      {...baseBuild, id: '11', hash: '1244', commitMessage: 'build 12', runAt: runAt(10)},
```
Comment (patrickhulce) [inline packages/server/test/test-utils.js:96]:
```suggestion
      {...baseBuild, id: '10', hash: '1245', commitMessage: 'build 11', runAt: runAt(11)},
      {...baseBuild, id: '11', hash: '1246', commitMessage: 'build 12', runAt: runAt(12)},
```

**Review by patrickhulce — APPROVED** [review]
thanks for going the extra mile @mattdell! I think you'll need to commit the image snapshots from storyshots that get generated but this LGTM with the changes :)

**Comment by mattdell** [comment]
Yeah, struggling with the snapshots...! For some reason when I run them locally I get around 30-40 failures. 🤔

**Comment by patrickhulce** [comment]
@mattdell I can handle the snapshot updates if you want to revert down to the package/yarn changes so I can push it out today. I'll cherrypick your commits as a base to still get ya the extra cred :)

**Comment by mattdell** [comment]
Thanks @patrickhulce!

For what it's worth it seems part of the problem is that I'm working within a fork of the original.

Screenshot 2021-04-12 at 21 36 54

As for the other ones... 🤷🏻‍♂️

**Comment by patrickhulce** [comment]
thanks @mattdell 🎉

**Pull request merged**